src/api/v1/__init__.py

```
@@ -99,7 +99,7 @@ def open(self):
             token = None
             try:
                 token = jwt.decode(encoded_token, self.settings['secret'], algorithm='HS256')
-            except jwt.DecodeError as jwt_error:
+            except Exception as jwt_error:
```

The decode method raises a DecodeError if there is a validation error but also can raise InvalidIssuedAtError, ImmatureSignatureError, ExpiredSignatureError, InvalidIssuerError and InvalidAudienceError if some of those claims fails when validating. Currently we are using only the "exp" claim so a ExpiredSignatureError can be raised. I catch all Exceptions to prevent future implementations of token claims as, for the moment, we do not need to distinguish between them.